applications/rogue/s3-iam-policy.json.tpl

```diff
@@ -0,0 +1,16 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "s3:*"
+            ],
+            "Resource": [
+                "${bucket_arn}/*",
+                "${bucket_arn}"
+            ]
+        }
+    ]
+}
```

Comment:
This is a simple copy-paste from the [`s3_bucket`](https://github.com/DoSomething/infrastructure/blob/main/components/s3_bucket/iam-policy.json.tpl) module (because each module only has access to it's own folder scope).